src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-ProfileCard/styles.ts

```diff
@@ -0,0 +1,180 @@
+import styled from '@emotion/styled';
+import * as C from '../../ignitus-Atoms/colors';
+import { flexibleColDiv, flexibleRowDiv, TextIconContainer } from '../../shared';
+import ChatMessageIcon from '../../ignitus-Assets/ignitus-Icons/chatMessageIcon/chatMessageIcon';
+import FileIcon from '../../ignitus-Assets/ignitus-Icons/fileIcon/fileIcon';
+import BookmarkIcon from '../../ignitus-Assets/ignitus-Icons/bookmarkIcon/bookmarkIcon';
+import MapMarkerIcon from '../../ignitus-Assets/ignitus-Icons/mapMarkerIcon/mapMarkerIcon';
+import { Heading1, Heading4 } from '../../ignitus-Atoms/typography';
+
+export const ProfileDiv = styled(flexibleColDiv)`
+  padding: 2rem;
+  margin-top: 100px;
+`;
+
+export const ProfileContainer = styled(flexibleRowDiv)`
+  border-radius: 4px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  background-color: ${C.White};
+  color: ${C.IgnitusBlue};
+  margin: 0.5rem;
+  padding: 1rem;
+  padding-top: 3px;
+  width: 100%;
+  flex-wrap: wrap;
+`;
+
+export const ProfileTrack = styled.div`
+  flex: 1 100%;
+  text-align: end;
+`;
+
+export const BookmarkIconDiv = styled(TextIconContainer)`
+  padding-left: 0;
+  padding-right: 2em;
+  text-align: right;
+  svg {
+    left: initial;
+    right: 0;
+    font-size: 2em;
+    top: -5px;
+  }
+`;
+
+export const ContentDiv = styled(flexibleRowDiv)`
+  padding: 0.5rem;
+  border: 2px solid blue;
+  flex: 1 60%;
+  flex-wrap: wrap;
+  border: none;
+`;
+
+export const ImageContainer = styled(flexibleRowDiv)`
+  width: 150px;
+  height: 150px;
+  img {
+    width: 100%;
+    height: 100%;
+    border-radius: 10px;
+  }
+`;
+
+export const DetailsContainer = styled.div`
+  flex: auto;
+  padding: 0.5em 0 0 1.5em;
+`;
+
+export const ProfileHeading = styled(Heading1)`
+  flex: 1;
+`;
+
+export const ProfileSubHeading = styled(Heading4)`
+  flex: 1;
+  margin-bottom: 10px;
+`;
+
+export const SocialDiv = styled(flexibleColDiv)`
+  padding: 0.5rem;
+  border: 2px solid blue;
+  flex: 1 40%;
+  border: none;
+`;
+
+export const SocialIcons = styled.div`
+  flex: 100%;
+  margin: 1em 0;
+  i {
+    color: ${C.IgnitusBlue};
+    font-size: 2em;
+    margin-right: 0.7em;
+  }
+`;
+
+export const TextIconContent = styled.span`
+  color: ${C.IgnitusBlue};
+`;
+
+export const SocialContact = styled(flexibleColDiv)`
+  flex: 100%;
+  p {
+    flex: 1 1 100%;
+  }
+`;
+
+export const ProfileBookmarkIcon = styled(BookmarkIcon)`
+  height: 2rem;
+  fill: ${C.IgnitusBlue};
+`;
+
+export const SendMessageIcon = styled(ChatMessageIcon)`
+  height: 1.5rem;
+`;
+
+export const ResumeIcon = styled(FileIcon)`
+  height: 1.5rem;
+`;
+
+export const LocationIcon = styled(MapMarkerIcon)`
+  height: 1.2rem;
+  position: relative;
```

Comment:
I still don't why you always use position property at all everything can be designed without using it at all, it makes sense when you are planning to use `z-index.`

When removing `position:relative` distorts  the UI badly.

![Screenshot 2020-03-25 at 01 35 48](https://user-images.githubusercontent.com/13464678/77489983-e60ca100-6e39-11ea-9441-9bc3e41d3044.png)


You can align `icon` with the `text` in a very simple manner, by having a flex box defined as.

```
    display: flex;
    flex-direction: row;
```
```
  svg {
    left: 0;
    position: absolute;
    top: 0;
    font-size: 1em;
		fill: ${C.IgnitusBlue};
  }
```
along with removal of css that you have defined on `SVG`,     

![Screenshot 2020-03-25 at 01 37 53](https://user-images.githubusercontent.com/13464678/77489992-eb69eb80-6e39-11ea-9417-9a3ce1832e47.png)

Reply:
Icons are of different sizes , therefore to align the text with those icons in single row I use position. By positioning icons fix the icon with the applied padding to the text.

Reply:
but don't you know you can change the size of icons? apart from that form where did you learnt to prefer position css property instead changing icon sizes?